src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -91,6 +91,7 @@ class Referral(
 
   @OneToOne(mappedBy = "referral") @Fetch(JOIN) var endOfServiceReport: EndOfServiceReport? = null,
   @OneToOne(mappedBy = "referral") @Fetch(JOIN) var supplierAssessment: SupplierAssessment? = null,
+  @OneToOne(mappedBy = "referral") @Fetch(JOIN) var referralLocation: ReferralLocation? = null,
```

Comment:
Do we know the expected performance cost of this new mapping? Asking as `Referral` is widely used across the codebase and now every materialisation of it would also make an extra join and allocate more memory